### PR TITLE
docs: add APISIX-integrates-with-Coraza.md

### PR DESCRIPTION
**Pull request by Yilialinn**
APISIX New Feature: APISIX integrates with Coraza

Fixes: #[Add issue number here]

Changes:

Screenshots of the change:

### CODE REVIEW

**Comment by Sn0rt** [comment]
LGTM

**Review by Sn0rt — APPROVED** [review]

**Review by SkyeYoung — DISMISSED** [review]

**Comment by kayx23** [comment]
Please correct typo `interagtes` in _file names_ and PR title.

**Review by SkyeYoung — DISMISSED** [review]

File: blog/en/blog/2023/09/08/APISIX-integrates-with-Coraza.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+    "nodes": {
+      "httpbin.org:80": 1
+    }
+  }
+}'
+```
+
+Test the WAF rules and we can see 403
+
+```
```
Comment (kayx23) [inline blog/en/blog/2023/09/08/APISIX-integrates-with-Coraza.md]:
```suggestion
```shell
```
same with the other ones with curl

File: blog/en/blog/2023/09/08/APISIX-integrates-with-Coraza.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+<center><h1>403 Forbidden</h1></center>
+<hr><center>openresty</center>
+<p><em>Powered by <a href="https://apisix.apache.org/">APISIX</a>.</em></p></body>
+</html>
+* Connection #0 to host localhost left intact
+```
+
+Check logs `logs/error.log`
+
+```
```
Comment (kayx23) [inline blog/en/blog/2023/09/08/APISIX-integrates-with-Coraza.md]:
```suggestion
```text
```

**Review by kayx23 — DISMISSED** [review]

**Review by SkyeYoung — APPROVED** [review]

**Review by kayx23 — APPROVED** [review]

**Pull request merged**